frontend/talentsearch/src/js/components/search/SearchContainer.tsx

```diff
@@ -68,7 +68,10 @@ export const SearchContainer: React.FC<SearchContainerProps> = ({
 
   function a(msg: string) {
     return (
-      <a href={`mailto:${poolOwner?.email}`} data-h2-font-weight="b(700)">
+      <a
+        href="mailto:recruitmentimit-recrutementgiti@tbs-sct.gc.ca"
```

Comment:
Good suggestion, this might be something we re-use. I'm not super familiar with how we are handling env for the frontend. Are you able to point me in the right direction for that change?

Reply:
Sure thing, have a look at [this file](https://github.com/GCTC-NTGC/gc-digital-talent/blob/main/frontend/talentsearch/src/js/talentSearchConstants.ts) for some examples used in frontend. I am happy to review the changes when you are ready or if you get stuck, also happy to run through it with you on a call.